zerver/tests/test_templates.py

```diff
@@ -197,3 +197,14 @@ def test_custom_tos_template(self):
         self.assert_in_success_response([u"Thanks for using our products and services (\"Services\"). ",
                                          u"By using our Services, you are agreeing to these terms"],
                                         response)
+
+    def test_help_guides(self):
+        root = '/home/vagrant/zulip/templates/zerver/help'
+        urls = list()
+        for file in os.listdir(root):
+            if file.endswith('.md'):
+                urls.append('/help/'+file.replace('.md', ''))
+        test_message = 'Expected 200, Received %d for %s'    
+        for url in urls:
+            response = getattr(self.client, 'get')(url) 
+            self.assertEqual(response.status_code, 200, msg = test_message %(response.status_code, url))    
```

Comment:
this is going to have some lint failures; please check out our coding style and adjust your code accordingly.